language/movie_nl/main_test.py

```diff
@@ -69,18 +69,18 @@ def test_process_movie_reviews():
     entities = [json.loads(entity) for entity in entities]
 
     # assert sentiments
-    assert sentiments[0].get('sentiment') == 1.0
+    assert sentiments[0].get('sentiment') == 0.9
```

Comment:
Can't these values change as the model changes? Should you just assert that sentiments are some non-None value?